Newsletter june (#146)
conda-incubator/conda-dot-org: blog/2023-06-20-newsletter.mdx

---
title: "Conda News, Issue #1!"
slug: "2023-06-20-newsletter"
description: |
    Recent and upcoming activity in the conda community.
authors: [tnabtaf, pinak-datta]
tags: [conda, newsletter]
image: img/blog/2023-06-20-newsletter/balloon-launch.jpg
---

import Image from '@theme/IdealImage';
import launch from '@site/static/img/blog/2023-06-20-newsletter/balloon-launch.jpg';
import scipybanner from '@site/static/img/blog/2023-06-20-newsletter/scipy-banner.png';
import packagingcon from '@site/static/img/blog/2023-06-20-newsletter/packagingcon.jpg';
import rattler from '@site/static/img/blog/2023-06-20-newsletter/rattler.png';
import youtube from '@site/static/img/blog/2023-06-20-newsletter/youtube.png';

<Image img={launch} alt="Banner image for blog post; The Conda Newsletter is launching - hot air balloon inflating"/>

<p>
    <i>
        <small>
            Image credit: <a href="https://unsplash.com/photos/sN40hKv8_cI">Road Trip with Raj on Unsplash</a>
        </small>
    </i>
</p>

> **Welcome to the first ever conda community newsletter!**  The newsletter will highlight recent and upcoming activity that is relevant to the [conda community](https://conda.org/).  This first issue highlights releases, grants, events, ....

Author: Oh you young people!  I will make a pass through the entire thing at some point to remove them.  They don't render any differently in the output.

> **Welcome to the first ever conda community newsletter!**  The newsletter will highlight recent and upcoming activity that is relevant to the [conda community](https://conda.org/).  This first issue highlights releases, grants, events, ....

Reviewer: If it doesn't affect the output, then feel free to ignore my used-to-modern-kerning self. :D

> We are hoping these will become a semi-regular feature in the conda landscape going forward.  If you have an item to include in a future newsletter please let us know by posting it to the [Conda Communications Matrix Channel](https://matrix.to/#/%23conda-communications:matrix.org) or by creating an issue in the [Conda Communications GitHub Repo](https://github.com/conda/communications/issues).

Author: I don't know either.  The name is CCF, but is it cCF when it's in the middle of sentence?

<!-- truncate -->

Finally, if you want to receive these newsletters from the comfort of your inbox, **please sign up for the [Conda Announce Mailing list](https://lists.conda.org/wws/info/announce).**

Happy reading,
The Editors

## Conda Community Gathering Spaces!

---

**This section will be replaced with a link to Pinak's writeup.**

---

<!--
The past year has seen the launch of a cohort of places for the conda community to gather online.  Please join us:

* **[conda.org](https://conda.org)**: The conda community now has it's own [web site](https://conda.org).

> * **[Q&A Forum](https://conda.discourse.group/)**: Q & about the entire ecosystem.

Author: Will be resolved in my next commit.

* **[Chat](https://matrix.to/#/#conda:matrix.org)**: Chat rooms about [everything](a), [conda-forge](a), [bioconda](a) [Constructor](a) and [Installer](a), [conda-build](a), [mamaba](a), [libmamba](a), ([Governance](a), [Communication](a)) and more.
* **[Conda Announce Mailing List](https://lists.conda.org/wws/info/announce)**: get monthly (or fewer) notifies about important conda stuff in your inbox (like this newsletter).
-->

## Community calls

> Please join us every other week to learn what is going on and to provide your input.

Author: Fixing now.

* [How to use conda-build to build a Python package with C/C++ dependencies](https://conda.org/blog/2023-05-18-how-to-use-conda-build), blog post by Vasvi Sood
* [How we reduced conda's index fetch bandwidth by 99%](https://conda.org/blog/2023-05-05-how-we-reduced-the-conda-index-fetch-bandwidth), blog post by Daniel Holth
* [Conda survey results](https://conda.org/blog/2022-03-30-conda-survey/), blog post by Dave Clements summarizing last year's survey of conda users, integrated with relevant bits from the PyPA's survey
* The [Conda Community Forum](https://conda.discourse.group/) saw navigation improvements with a new sidebar for navigating the site and a new top menu for navigating to other conda websites.

## Conda, now starring on YouTube

**Conda now has a [YouTube channel](https://www.youtube.com/channel/UCNc3BL-gO8hbx_7DdEDcofg)** with four playlists/videos to start you off: **[Packaging](https://youtu.be/6by8b2Ht200), [Presentations](https://youtu.be/OeSv0_HesaU), [Training](https://youtu.be/NSgTFf40vHQ)**, and **[Releases](https://youtu.be/VD5TXcbYgR4)**.

<Image img={youtube} alt="Montage showing screenshots from 4 videos."/>

> Or, a long version.  I prefer the short one.

Author: That's enough for me.  Yanking long version in next commit.

* [Releases](https://youtu.be/VD5TXcbYgR4): featuring the *Conda 22.11 Release Video*, by the contributors, December 2023.

## Releases and Announcements

* **[conda-lock 2.0.0](https://github.com/conda/conda-lock/releases/tag/v2.0.0)** - a lightweight library for generating fully reproducible lock files for conda environments.
* **[conda 23.5.0](https://conda.org/blog/2023-05-31-may-2023-releases#changes-in-conda)** - Python 3.11 support and the new [**conda doctor** subcommand plugin](https://conda.org/blog/2023-06-01-conda-doctor) for detecting corrupted installs.
* **[conda-build 3.25.0](https://conda.org/blog/2023-05-31-may-2023-releases#changes-in-conda-build)** - an update to `conda`'s package builder.
* **[conda-libmamba-solver 23.5.0](https://conda.org/blog/2023-05-31-may-2023-releases#changes-in-conda-libmamba-solver)** - An update to the fast `libmamba` solver in `conda`.
* **[grayskull 2.3.1](https://github.com/conda/grayskull/releases/tag/v2.3.1)** - An automatic recipe generator for the conda ecosystem.
* **[Prefix.dev Channels](https://prefix.dev/blog/introducing_channels)** - A new option for creating your own conda channels.

> * **[conda-package-streaming 0.8.0 & conda-package-handling 2.1.0](https://conda.discourse.group/t/conda-package-streaming-0-8-0-and-conda-package-handling-2-1-0-released/267)** - read new & old formats (`.conda & .tar.bz2`) and create & extract conda packages.

Reviewer: ```suggestion
* **[conda-package-streaming 0.8.0 & conda-package-handling 2.1.0](https://conda.discourse.group/t/conda-package-streaming-0-8-0-and-conda-package-handling-2-1-0-released/267)** - Read new & old formats (`.conda & .tar.bz2`) and create & extract conda packages.
```
This is also the only item in this list whose explanation starts with a verb. Maybe "Updates for reading new and old formats ... and creating and extracting conda packages."?

-->

### Community calls

Please join us every other week to learn what is going on in the conda community and to provide your input.

* The Mamba project has [bi-weekly community calls](https://hackmd.io/iOgkxNMpTea6lWczjCov9Q?view) on Tuesdays. Our next call is June 27.
* The overall conda community has [bi-weekly calls](https://hackmd.io/@conda-community?tags=%5B%22meeting-notes%22%5D) on Wednesdays. The next one is on June 21.

## Upcoming events

### SciPy 2023 Conda & Friends Sprint, July 15-16, Austin

<Image img={scipybanner} alt="Banner image for SciPy2023"/>

The [conda community will again be meeting] **(LINK TO MATRIX CHANNEL?)** at the [SciPy 2023](https://www.scipy2023.scipy.org/) meeting in Austin, Texas for two days of collaborative work.

[Sprints](https://www.scipy2023.scipy.org/sprints) are open to everyone who is interested in contributing to the various projects in the conda ecosystem (e.g. projects under the [conda](https://github.com/conda), [conda-incubator](https://github.com/conda-incubator), [mamba-org](https://github.com/mamba-org), [conda-forge](https://github.com/conda-forge), ... GitHub organizations), what we call "conda and friends".

> First and foremost, **the goal of conda community sprints is to grow the conda contributor community**. We mean "contributor" in the broadest possible sense: not just code, but also documentation, bug reports, forum questions (and answers), community events, new tools and plugs, and of course, new pull requests for existing tools too.

Reviewer 2: ```suggestion
First and foremost, **the goal of conda community sprints is to grow the conda contributor community**. We mean "contributor" in the broadest possible sense: not just code, but also documentation, bug reports, forum questions (and answers), community events, new tools and plugins, and of course, new pull requests for existing tools too.
```

* [**rattler-build**](https://prefix.dev/blog/the_new_rattler_build) - a new rewrite of the core concepts of conda-build in *Rust*.

## Community channel updates

### conda-forge

> **conda-forge has a Google Summer of Code project to update the [project website](https://conda-forge.org/)**. [Asmit Malakannnawar](https://asmitbm.github.io/) will be working on the update for the next few months.  The update will provide an intuitive and usable user experience that also passes SpeedPage and [WAVE](https://wave.webaim.org/) tests, and that moves the site closer to [WCAG 2](https://www.w3.org/WAI/standards-guidelines/wcag/) conformance.  He'll also produce a site style guide.  Have ideas? [Reach out to Asmit](https://asmitbm.github.io/assets/html/contact.html).

Reviewer 3: ```suggestion
**conda-forge has a Google Summer of Code project to update the [project website](https://conda-forge.org/)**. [Asmit Malakannnawar](https://asmitbm.github.io/) will be working on the update for the next few months.  The update will provide an intuitive and usable user experience that also passes SpeedPage and [WAVE](https://wave.webaim.org/) tests, and that moves the site closer to [WCAG 2](https://www.w3.org/WAI/standards-guidelines/wcag/) conformance.  He'll also produce a site style guide.  Have ideas? [Reach out to Asmit](https://asmit.bio.link/).
```
